test/integration/about_page_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class AboutPageTest < ActionDispatch::IntegrationTest
+  def assert_about_page_i18n(local)
+    get page_url("about"), locale: local
+
+    assert_response :success
+    assert_select(".t-body a") do |elements|
+      assert_equal 13, elements.count
+    end
+  end
+
+  test "about page i18n for all supported languages" do
+    assert_about_page_i18n 'en'
+    assert_about_page_i18n 'es'
+    assert_about_page_i18n 'de'
+    assert_about_page_i18n 'nl'
+    assert_about_page_i18n 'fr'
+    assert_about_page_i18n 'pt-BR'
+    assert_about_page_i18n 'zh-CH'
+    assert_about_page_i18n 'zh-TW'
```

Comment:
You can get the list from `I18n.available_locales` so we don't need to hardcode the available locales here.